client/landing/gutenboarding/section.ts

```diff
@@ -0,0 +1,8 @@
+export const GUTENBOARDING_SECTION_DEFINITION = {
+	name: 'gutenboarding',
+	paths: [ '/gutenboarding' ],
+	module: 'gutenboarding',
+	secondary: false,
```

Comment:
`secondary: false` might not be needed here since we don't have Calypso `Layout`?